ninjabot.go

```diff
@@ -261,6 +262,10 @@ func (n *NinjaBot) backtestCandles() {
 	// to avoid sync issues between multiple coins
 	n.startBacktest.Wait()
 
+	//load maximum of progresbar by using candle queue
+	maxProgressbar := n.priorityQueueCandle.Len()
+	bar := progressbar.Default(int64(maxProgressbar))
```

Comment:
Just a small change.
I think that initialization may be placed before `n.startBacktest.Wait()` (line 263)
While ninjabot reads the CSV input, it will be ready in the `0%` state.

Reply:
hi, please check again I initial progress bar before feed data
(I tested by using 500k data but it's very fast xD)